libs/components/datetime/src/lib/modules/datepicker/datepicker-input-fuzzy.directive.ts

```diff
@@ -604,4 +533,73 @@ export class SkyFuzzyDatepickerInputDirective
   private onTouched = () => {};
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   private onValidatorChange = () => {};
+
+  private updateValue(value: any, isProgramatic = false): void {
```

Comment:
Spelling: `isProgrammatic`.

Reply:
Also, what is meant by "programmatic"? As in, the value change is done _internally_ to the control, and not as a result of a user? If there's not a better word to describe the behavior we should put a JSDoc comment above this method explaining what the `isProgrammatic` argument does.

Reply:
The `isProgrammatic` denotes that it was done via a `setValue` call to the form control or internally. If this is the case - we don't want to fire an `onChange` event as that triggers `valueChange` and `statusChange` and extra time.

Comment added